cp-attendance-hybrid/src/navigation/Routes.tsx

```diff
@@ -4,90 +4,174 @@ import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 import { Home } from '@src/modules/home/home.page';
 import { Theme } from 'assets/theme/theme';
 import { Menu } from '@src/modules/menu/menu.page';
-import { View } from 'react-native';
+import { GestureResponderEvent, StyleSheet, TouchableOpacity, View } from 'react-native';
 import Icon from 'react-native-vector-icons/Entypo';
 import { Schedule } from '@src/modules/schedule/schedule.page';
 import { Profile } from '@src/modules/profile/profile.page';
+import { useEffect, useRef } from 'react';
+import * as Animatable from 'react-native-animatable';
 
 export type RootStackParamsList = {
   Home?: undefined;
   Profile?: undefined;
+  Menu?: undefined;
+  Schedule?: undefined;
+  Tabs?: undefined;
 };
 
+interface TabButtonProps {
+  label: string;
+  icon: string;
+  onPress: ((e: GestureResponderEvent) => void) | undefined;
+  selected: boolean;
+}
+
+const TabArr = [
+  { route: 'Home', label: 'Home', icon: 'home', component: Home },
+  { route: 'Menu', label: 'Menu', icon: 'menu', component: Menu },
+  { route: 'Schedule', label: 'Grade', icon: 'calendar', component: Schedule },
+];
+
 export type ScreenProps<T extends keyof RootStackParamsList> =
   NativeStackScreenProps<RootStackParamsList, T>;
 
 const Stack = createNativeStackNavigator<RootStackParamsList>();
+const Tab = createBottomTabNavigator<RootStackParamsList>();
+
+const animate1 = { 0: { scale: 0.5, translateY: 7 }, 0.7: { translateY: -34 }, 1: { scale: 1.2, translateY: -24 } };
```

Comment:
Acho que seria legal colocar um nome mais descritivo, tipo, o que a animação faz (deslizar pro lado, pular na tela, sla q krl) pq se algum de nois meros mortais do web dev for mexer eu n vou saber oqq faz n sipa sem perguntar pro copilot (talvez pq eu n manje de front mas blz)